src/views/Register.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import { Redirect } from 'react-router-dom';
+import PropTypes from 'prop-types';
+
+const handleRegister = (event, registerUser) => {
+  event.preventDefault();
+  let name = '';
+  let hotelId = '';
+  let password = '';
+  let email = '';
+
+  event.target.parentNode.childNodes.forEach((childNode) => {
+    if (childNode.name === 'name') {
```

Comment:
That `if` block is a good candiate for `switch` statemants with cases, but that just one's preference.